packages/manager/src/components/SelectionCard/SelectionCard.spec.js

```diff
@@ -1,205 +1,246 @@
 const { navigateToStory , executeInAllStories } = require('../../../e2e/utils/storybook');
 
 describe('Selection Card Suite', () => {
```

Comment:
it might be good to check with the team to see if repeating some of these tests across every story makes sense, or if the same underlying functions power all selection cards (like running `assertCardsChecked` as many times as we do).. 

These tests are so cheap though, it's not a big issue, more of a consideration.

Reply:
I would like to get more from them on some of this as well. I agree though they are cheap so not that big of a deal.